### PR TITLE
Add policy file info

### LINKED ISSUE #239: Update developer docs to manually install unlimited-strength JCE
Until #9 is fixed, the developer docs have to be updated to manually install unlimited-strength JCE in order to be able to decrypt encrypted forms.

### PR DESCRIPTION
**Pull request by yanokwa**
Closes #239

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,7 @@ To package a runnable jar, use the `jar` Gradle task.
 
 To try the app, you can use the demo server. In the window that opens when running, choose Connect, then fill in the URL [http://opendatakit.appspot.com](http://opendatakit.appspot.com) leave username and password blank.
 
+
```
Comment (rockydcoder) [inline README.md]: Why is this extra blank line added?

File: README.md [hunk 2]
```diff
@@ -30,7 +30,13 @@ The main class is `org.opendatakit.briefcase.ui.MainBriefcaseWindow`. This repos
 - `org.opendatakit.briefcase.ui.MainClearBriefcasePreferencesWindow` clears Briefcase preferences
 - `org.opendatakit.briefcase.ui.MainFormUploaderWindow` uploads blank forms to Aggregate instances
 
-Note: There might be some compile errors in the IDE about a missing class `BuildConfig`. That class is generated by Android Tools at runtime. The warnings can be ignored.
+There might be some compile errors in the IDE about a missing class `BuildConfig`. That class is generated by gradle and the warnings can be ignored.
+
+If are working with [encrypted forms](http://docs.opendatakit.org/encrypted-forms) you may get a `InvalidKeyException`. This is because you do not have an unlimited crypto policy enabled in Java. Do this:
```
Comment (rockydcoder) [inline README.md]: I think it should be _**an** `InvalidKeyException`_

**Comment by yanokwa** [comment]
Thanks so much for the thorough review, @rockydcoder!

@lognaturel this is ready to merge!

**Comment by lognaturel** [comment]
@rockydcoder you're the best!

**Pull request merged**